packages/react-components/react-tree/src/components/TreeItem/useTreeItem.tsx

```diff
@@ -38,7 +37,35 @@ export const useTreeItem_unstable = (props: TreeItemProps, ref: React.Ref<TreeIt
       iconAfter: 'span',
       actions: 'span',
       badges: 'span',
+      groupper: 'span',
+    },
+    root: {
+      ...treeItemState.root,
+      onClick: useEventCallback(event => {
+        //  if click event comer from actions, ignore it
+        if (actionsRef.current && actionsRef.current?.contains(event.target as Node)) {
```

Comment:
you could use the `elementContains` utility that we have as a part of react-portal, as long as the portal element is rendered by the Fluent portal, you should have the same result

https://github.com/microsoft/fluentui/blob/696615b6a369e288a44ffa70f212593057709e6e/packages/react-components/react-portal/src/virtualParent/elementContains.ts#L8-L28